src/Main.js

```diff
@@ -24,3 +24,6 @@ export { default as GeoJSON2Features } from './Renderer/ThreeExtended/GeoJSON2Fe
 export { default as FeaturesUtils } from './Renderer/ThreeExtended/FeaturesUtils';
 export { CONTROL_EVENTS } from './Renderer/ThreeExtended/GlobeControls';
 export { default as DEMUtils } from './utils/DEMUtils';
+// Export for test
+// Question: how export TileMesh only for the test?
+export { default as TileMesh } from './Core/TileMesh';
```

Comment:
How export `TileMesh` only for the test?
I modify `TileMesh` function ([here](https://github.com/iTowns/itowns/pull/538/files#diff-7c875b4c6fb106f12afe62df279296c5R7)) only for the test to not pollute the code and use.

Reply:
You can import TileMesh in tests without adding it in Main.js, you just need the correct path, see  #594 for instance (and the existing unit tests).

Reply:
And in #594 you have also some functions exported only for testing.

Reply:
@autra thanks!